examples/templates/kubevirt/cluster.tf

```diff
@@ -180,13 +180,37 @@ resource "kubernetes_manifest" "kubevirtmachinetemplate_control_plane" {
             }
             "spec" = {
               "runStrategy" = "Always"
+              "dataVolumeTemplates" = [
+                {
+                  "metadata" = {
+                    "name" = "vmdisk-dv"
+                  }
+                  "spec" = {
+                    "pvc" = {
+                      "accessModes" = ["ReadWriteOnce"]
+                      "resources" = {
+                        "requests" = {
+                          "storage" = "50Gi"
+                        }
+                      }
+                    }
+                    "source" = {
+                      "http" = {
+                        "url" = "https://www.talos.pair.sharing.io/ubuntu-2004-kube-v1.23.10.gz"
```

Comment:
@BobyMCbobs I'm having trouble downloading these, may need to look into providing another in-cluster way of serving up large files.